finetune/encoding/input_encoder.py

```diff
@@ -170,50 +169,18 @@ def encode_multi_input(self, Xs, Y=None, max_length=None, pad_token=None):
         :param max_length: Max length of the sequences.
         :return: A Labeled Sequence Object.
         """
-        token_ids = []
-        tokens = []
-        positions = []
-        labels = []
-
-        # for each field in that example
-        for field in Xs:
-            assert isinstance(
-                field, (list, tuple)
-            ), "This should be a list of strings, instead it's {} {}".format(
-                tf.contrib.framework.nest.map_structure(type, field), Xs
-            )
-            encoded = self._encode(field, labels=Y)
-            token_ids.append(_flatten(encoded.token_ids))
-            tokens.append(_flatten(encoded.tokens))
-            positions.append(_flatten(encoded.char_locs))
-            labels.append(_flatten(encoded.labels))
-            if len(tokens[-1]) > (max_length - 2):
-                warnings.warn(
```

Comment:
Would like to preserve this warning if opssible.

Reply:
There was a redundant warning in cut and concat. I have added in the bit about try chunk_long_sequences